test/apps/iodemo/io_demo.cc

```diff
@@ -1316,6 +1316,39 @@ class DemoClient : public P2pDemoCommon {
         return data_size;
     }
 
+    void
+    get_server_info_uncompleted_ops(const server_info_t& server_info,
+                                    std::stringstream &ss) const
+    {
```

Comment:
unrelated?

Reply:
yes, started re-working this PR on top of another PR by mistake.
fixed